locales/en-US/webextension.properties

```diff
@@ -6,6 +6,7 @@ contextMenuLabel = Take a Screenshot
 myShotsLink = My Shots
 screenshotInstructions = Drag or click on the page to select a region. Press ESC to cancel.
 saveScreenshotSelectedArea = Save
+saveScreenshotSelectedAreaToCloud = Save to Cloud
```

Comment:
`make clean && make addon` wasn't enough to make this new string appear, for some odd reason

Reply:
Uhm, what's "Cloud" in this context? I feel like we should add an explanation in a comment